src/main/java/org/openpnp/machine/reference/vision/wizards/ReferenceFiducialLocatorConfigurationWizard.java

```diff
@@ -0,0 +1,233 @@
+package org.openpnp.machine.reference.vision.wizards;
```

Comment:
I find the template option confusing. I think this needs to be simplified a bit too. We may need to discuss this further, but here is my thinking:

* The code that processes the pipeline should detect a stage called "template" and add the template. I think this is happening now, so that's good.
* Instead of having distinct code paths for template matching and blob detection, which limits the ability to customize the pipeline, just specify that certain result stages will be processed in different ways. i.e. if after processing, the pipeline contains a stage called "resultTemplates", process the results as template matches. If it contains a stage called "resultKeypoints" process the results as keypoints. Even better would be to just look for "results" and make a decision based on it's data type.

In general, try to present as few options as possible and do as much for the user as you can.